azure-pipelines.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       addToPath: true
   - bash: |
       python -m pip install --upgrade pip
-      python -m pip install numpy scipy matplotlib sphinx pytest pytest-cov pytest-timeout pytest-sugar flake8
+      python -m pip install numpy scipy matplotlib sphinx pytest pytest-cov pytest-timeout pytest-sugar flake8 flake8-import-order pep8-naming
```

Comment:
I would not add this to our CIs. We already enforce many things to our users.

Reply:
On second thought if we aren't going to check with CI then we should just ignore all things. So let's just drop this effort entirely for now and someone can pick it up later if we can converge on the goals